hello-nextflow/hello-config.nf

```diff
@@ -7,18 +13,13 @@ params.greeting = 'greetings.csv'
 params.batch = 'test-batch'
 params.character = 'turkey'
 
-// Include modules
-include { sayHello } from './modules/sayHello.nf'
-include { convertToUpper } from './modules/convertToUpper.nf'
-include { collectGreetings } from './modules/collectGreetings.nf'
-include { cowpy } from './modules/cowpy.nf'
-
 workflow {
 
     // create a channel for inputs from a CSV file
-    greeting_ch = Channel.fromPath(params.greeting)
-                        .splitCsv()
-                        .map { line -> line[0] }
+    greeting_ch = Channel
```

Comment:
I'm not sure I like having to put the `.fromPath()` bit on the next line. I would keep that like it was before tbh. That way you have the first line do a full thing, then the next lines modify that thing. If you have just `greeting_ch = Channel` on one line that feels... interrupted

Reply:
For me I would do this for any old `foo.bar()` as well I think.. The first line should contain the first action. Whether it's `Channel` or `foo`, neither of these are actions, only the starting points.

Reply:
When I imagine something like this:

Reply:
When I imagine something like this:

```groovy
ch_samples
  .filter { sample -> sample.n_reads >= 1_000_000 }
  .map { sample -> sample.id }
```

I think I prefer both calls to be on their own line. I think I treat channel factories differently in my head because the `Channel` and the factory call are sort of a unit, but in general it seems like the "starting point" should be separate from any transformations performed on it

Reply:
the `format` command will now put `Channel.fromPath()` on a single line, as well as any other channel factory